### ISSUE TITLE
Replacing `request` with `rock-req` breaks the library

### ISSUE THREAD

**Issue by dschoeni** [node-ebics/node-ebics-client#90]
I believe the API of `rock-req` is not followed correctly, and this breaks the library in its current release (v4)

https://github.com/node-ebics/node-ebics-client/blob/master/lib/Client.js#L218

```
rock.post(
  this.url, 
  {
     body: r,
     headers: { 'content-type': 'text/xml;charset=UTF-8' },
  },
...
```

Should be replaced with

```
rock.(
  {
    method: 'POST',
    url: this.url, 
     body: r,
     headers: { 'content-type': 'text/xml;charset=UTF-8' },
  },
...
```

See https://github.com/carboneio/rock-req/blob/master/doc/api.md

This also seems to break the response parsing in the lib.

**Comment by nanov** [comment]
Thank you so much for reporting, will provide a fix now and cover with integration tests asap!

**Comment by nanov** [comment]
Better late than never, addressed in 4.1.0
